frontend/src/js/components/Uploads/UploadTarget.ts

```diff
@@ -44,11 +37,10 @@ export async function getUploadTarget(
         throw new Error(`No workspace entry in either focused entry ${focusedEntry} or root of workspace ${workspace.id}, cannot upload to workspace`);
     }
 
-    const ingestionUri = await createUploadIngestion(defaultCollection);
-
+    const ingestionName = `Upload ${new Date().toISOString()}`; 
```

Comment:
non-blocking: while we're creating this new endpoint, would it be worth simplifying things by have the api create the ingestion name based on the current datetime?
Seems like the advantage might be that we simplify client-side code and reduce the risk of misnamed ingestions due to the client's clock being incorrect, and instead use the more reliable server time.

Reply:
Yes this is a great idea. To do this there needs to be a bit more changes in the backend, and I think it might be better if that is not included part of this PR to reduce the risk of introducing bug. if that's OK with you, I created this health ticket for it https://trello.com/c/tRaxMQI3/2881-giant-simply-client-by-creating-the-ingestion-name-in-backend